### DISCUSSION TITLE
No success with setting up quasar on stackblitz

### DISCUSSION THREAD

**Discussion by rowild** [quasarframework/quasar#14025] [General - Components / Directives / etc] [answered]
This project
https://stackblitz.com/edit/quasarframework-enyltx?file=src%2Flayouts%2FMainLayout.vue
throws errors in the console like 
`Uncaught (in promise) TypeError: Failed to fetch dynamically imported module: https://quasarframework-enyltx--9000.local.webcontainer.io/src/boot/axios.js`
But axios is called from `boot/axios.js` in `quasar.config.js`

There are also lots and lots of warning concerning the loading of maps from node_modules... Do you experience the same things?

And sometimes i get
`Failed to execute 'postMessage' on 'DOMWindow': The target origin provided ('https://quasarframework-enyltx--9000.local.webcontainer.io') does not match the recipient window's origin ('https://stackblitz.com').`

(`https` is set to false on quasar.config)

I desperately need this working since I would actually like to ask something about quasar and therefore need a little test project. Any help would be really much appreciated!

**Comment by yusufkandemir** [comment, marked as answer]
This must be about StackBlitz's behavior. When I open the browser console before the project is compiled and rendered, I either get a white screen or some errors similar to the ones you shared. This is not specific to Quasar. You must have figured a way out since you opened #14035. So, I think this is resolved now.

**Reply by rowild** [reply]
@yusufkandemir Thank you again! You are right, this issue is obsolete. I removed lots of stuff and I think there was also an error in the router file on my part. The warnings concerning not found maps still remain, though, but I turned them off via my browser console.
